scripts/clone-data-across-firebase-projects.ts

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import { resolve } from "path";
+
+import { AnyVenue } from "../src/types/Firestore";
+
+import { initFirebaseAdminApp } from "./lib/helpers";
+
+// ---------------------------------------------------------
+// Configuration (this is the bit you should edit)
+// ---------------------------------------------------------
+
+const SOURCE_PROJECT_ID = "co-reality-map";
+const DEST_PROJECT_ID = "co-reality-staging";
+
+const SOURCE_CREDENTIAL_FILE =
+  "co-reality-map-firebase-adminsdk-47j7x-530045073b.json";
+const DEST_CREDENTIAL_FILE =
+  "co-reality-staging-firebase-adminsdk-yy5cq-5fd568c2f4.json";
```

Comment:
Should these be command line args?

Reply:
Eventually, definitely. But I would say we should move to a proper CLI framework as per https://github.com/sparkletown/internal-sparkle-issues/issues/39 before doing so, as otherwise it's just going to be way too unruly, with too many arguments, etc.

Reply:
I kind of envision it the same way the table generation script works currently. We edit the config within the script as part of running it.

Reply:
Yep, makes sense.